tests/run-tests.py

```diff
@@ -999,9 +999,21 @@ def run_one_test(test_file):
             with open(filename_mupy, "wb") as f:
                 f.write(output_mupy)
             failed_tests.append((test_name, test_file))
+            if uses_unittest:
+                rm_f(filename_expected)  # in case left over from previous failed run
 
         test_count.increment()
 
+        # Print a note if this looks like it might have been a misfired unittest
+        if not (uses_unittest or test_passed):
```

Comment:
This logic was a bit hard for me to grok.  Maybe better as `if not uses_unittest and not test_passed`??

Reply:
Yeah... I paused on this when I wrote it, but wasn't sure which one reads cleaner. Have swapped to the other one!